controllers/front/index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* 2007-2014 PrestaShop
```

Comment:
Outdated header 😄 

You can automate your IDE to generate these

Reply:
![ohzrd5t](https://user-images.githubusercontent.com/1462701/45763665-3e555800-bc31-11e8-9b8a-d775b10cf689.jpg)